STYLEGUIDE.md

```diff
@@ -32,8 +31,10 @@ Assume the reader is a busy non-native English speaker.
 ### Referring to the Product and Features
 
 - Sourcegraph: main product, prefer using this name unless you need to be more precise
-  - Sourcegraph Server: the form of our product that ships as the `sourcegraph/server` Docker image and runs on a single node
-  - Sourcegraph Data Center: the form of our product that runs on Kubernetes
+  - Sourcegraph OSS: the open source code for Sourcegraph. This is **not** the same as Sourcegraph Core.
```

Comment:
open-source (with hyphen when used as an adjective)

Also I would not include Sourcegraph OSS in this list, as it's not really a product. You could mention it in a bullet point underneath and say something like "When referring to the build result of the open-source repository, use the name Sourcegraph OSS."